src/main/webapp/profile.html

```diff
@@ -144,289 +147,235 @@ <h1>TheGlobetrotter</h1>
-                  <span class="fas fa-star checked"></span>
-                  <span class="fas fa-star-half-alt"></span>
-                </div>
-              </div>
+        <div class="card">
+          <div class="container">
+            <div class="route-details">
+              <p class="name">Barajul Drăgan-Floroiu</p>
+              <p class="description">Some Basic info</p>
+            </div>
+
+            <img
+              class="route-img"
+              src="./pictures/routePhotos/eighth.jpg"
+              alt="Lunca Vişagului, Romania"
+            />
+
+            <div class="rating">
+              <span class="fas fa-star checked"></span>
+              <span class="fas fa-star checked"></span>
+              <span class="fas fa-star checked"></span>
+              <span class="fas fa-star checked"></span>
+              <span class="fas fa-star-half-alt"></span>
             </div>
           </div>
         </div>
       </div>
 
-      <div id="new-route" class="content-item">
-        <div class="row">
-          <div class="column">
-            <div class="card">
-              <div class="container">
-                <p>Create New Route</p>
-                <a href="index.html">
-                  <div href="index.html" class="new-route">
-                    <span class="fas fa-plus"></span>
-                  </div>
-                </a>
-              </div>
-            </div>
+      <!-- Favourite places -->
+      <div id="fav-places" class="content-item">
+        <div class="card">
+          <div class="toggle-popup" onclick="showFavPlaceDetails('place-id-1')">
+            <span class="fas fa-map-marker-alt"></span>
+            <div class="fav-place-desc">Place name 1</div>
+          </div>
+          <div class="popups" id="place-id-1-popup">
+            <p>First location name</p>
+            <p>Some info with links.</p>
           </div>
         </div>
+
+        <div class="card">
+          <div class="toggle-popup" onclick="showFavPlaceDetails('place-id-2')">
+            <span class="fas fa-map-marker-alt"></span>
+            <div class="fav-place-desc">Place name 2</div>
+          </div>
+          <div class="popups" id="place-id-2-popup">
+            <p>First location name</p>
+            <p>Some info with links.</p>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="toggle-popup" onclick="showFavPlaceDetails('place-id-3')">
+            <span class="fas fa-map-marker-alt"></span>
+            <div class="fav-place-desc">Place name 3</div>
+          </div>
+          <div class="popups" id="place-id-3-popup">
+            <p>First location name</p>
+            <p>Some info with links.</p>
+          </div>
+        </div>
+      </div>
+
+      <!-- Create new route -->
+      <div id="new-route" class="content-item">
+        <div class="card" onclick="location.href = '/index.html';"></div>
```

Comment:
Did you remove the plus sign on purpose?

Reply:
I just lost it 💦